test/serialize.cpp

```diff
@@ -440,3 +440,85 @@ TEST(serialize, opset1_pad)
     EXPECT_EQ(g_pad->get_version(), 1);
     EXPECT_EQ(dynamic_cast<const op::v1::Pad*>(g_pad.get())->get_pad_mode(), pad_mode);
 }
+
+enum class TuringModel
+{
+    XL400,
+    XL1200
+};
+
+namespace ngraph
+{
+    template <>
+    EnumNames<TuringModel>* EnumNames<TuringModel>::get()
```

Comment:
Why do you use raw pointer and don't return a reference to a local static object?

Reply:
Too late in the day when I wrote it 😄